Server/mods/deathmatch/logic/luadefs/CLuaDefs.cpp

```diff
@@ -10,6 +10,7 @@
  *****************************************************************************/
 
 #include "StdInc.h"
+#include <lua/LuaCommon.h>
```

Comment:
```suggestion
#include "lua/LuaCommon.h"
```
It's a regular header file